examples/agent_rtm_example.py

```diff
@@ -0,0 +1,20 @@
+''' Agent RTM client example usage. '''
+
+from livechat.agent.rtm.client import AgentRTM
+
+agent_rtm = AgentRTM.get_client()
+agent_rtm.open_connection()
+agent_rtm.login(token='Bearer <your bearer token>')
```

Comment:
```suggestion
agent_rtm.login(token='<your access token>')
```
I would change it like above, because it's also possible to provide `Basic` token (e.g. PAT).

Reply:
👍